### PR TITLE
[OSCD] T1070.004: File Deletion - Linux

### PR DESCRIPTION
**Pull request by omergunal**
https://github.com/Neo23x0/sigma/issues/1011

Task 14

### CODE REVIEW

**Comment by zinint** [comment]
Hello and welcome!

Please help me to understand, are you planning to detect those via audit.d with the rule that's smth like this: (?)
``` bash
-a exit,always -F arch=b64 -S execve -k auditcmd
```
Then you should probably use some additional `{field: value}`s in your detection, e.g.
``` YAML
service: auditd
detection:
    selection:
        type: 'EXECVE'
        a0: 'rmdir'
```
for example, like here [rules\linux\auditd\lnx_auditd_susp_cmds.yml](https://github.com/Neo23x0/sigma/blob/master/rules/linux/auditd/lnx_auditd_susp_cmds.yml) or here [rules\linux\lnx_chattr_immutable_removal.yml](https://github.com/Neo23x0/sigma/blob/master/rules/linux/lnx_chattr_immutable_removal.yml).

What do you think? Did you run some tests? What did they show?

**Comment by zinint** [comment]
Hello again @omergunal!

0. We had discussed this a bit and decided that your format of describing EXECVE type of detection is better, simply by using `Commands|contains` instead of using `a0`, `a1`, etc. So well done, let's leave it as you did. But you should add a `category` value `process_creation` to your rules.
1. `rules/linux/lnx_file_deletion.yml` - it covers the ART test and even more as I can see. So just add a `category`.
2. `rules/windows/registry_event/sysmon_comhijack_sdclt.yml` you already put this rule in PR #1078 , and I already commented on it [there](https://github.com/Neo23x0/sigma/pull/1078#issuecomment-707334832). Please decide where you'll leave it and delete the other one.

**Comment by zinint** [comment]
Thanks for the update!

I think last change we need is adding this link https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1070.004/T1070.004.md to the rule's `references` field

**Comment by zinint** [comment]
But you didn't do anything with the second rule in every one of your PRs `rules/windows/registry_event/sysmon_comhijack_sdclt.yml`:
1. Will you remake your PRs or leave it in everyone anyway?
2. Will you change the rule's `detection` according to the test wich [I described here](https://github.com/Neo23x0/sigma/pull/1078#issuecomment-707334832).

File: rules/linux/lnx_file_deletion.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+detection:
+    keywords:
+        - Commands|contains:
+          - 'rm '
+          - 'shred -u'
+          - 'rmdir'
+          - 'unlink'
+          - 'busybox rm -f *'
+          - 'find * -delete'
+    condition: keywords
```
Comment (yugoslavskiy) [inline rules/linux/lnx_file_deletion.yml:21]: Hello @omergunal! Could you please update the `detection` section using the approach that we discussed in the other PRs of yours?

**Review by yugoslavskiy — APPROVED** [review]
@thomaspatzke @Neo23x0 guys, I believe that this PR could be merged (:
please let us know what do you think about it if there is anything you would like to change.

**Pull request merged**